apps/playwright/utils/user.ts

```diff
@@ -0,0 +1,58 @@
+export const defaultUser = {
+  // required
+  firstName: "Playwright",
+  lastName: "User",
+  preferredLang: "EN",
+  preferredLanguageForInterview: "EN",
+  preferredLanguageForExam: "EN",
+
+  // optional
+  telephone: undefined,
```

Comment:
I think this stuff is a hangover from when Cypress was in plain JS.  It'd probably be better to just import the User type from graphql and type this object.

Reply:
Oh, thanks! I was kind of just copy and pasting a lot of stuff from Cypress to this so that is good to know 👍